ostelco-ios-client/ViewControllers/EKYC/PendingVerificationViewController.swift

```diff
@@ -81,14 +81,16 @@ class PendingVerificationViewController: UIViewController {
     func handleRegionPendingOrRejected(silentCheck: Bool = false, regionResponse: RegionResponse) {
         if let jumioStatus = regionResponse.kycStatusMap.JUMIO, let nricStatus = regionResponse.kycStatusMap.NRIC_FIN, let addressStatus = regionResponse.kycStatusMap.ADDRESS_AND_PHONE_NUMBER {
             switch (jumioStatus, nricStatus, addressStatus) {
-            case (let jumioStatus, let nricStatus, let addressStatus) where jumioStatus == .REJECTED || nricStatus == .REJECTED || addressStatus == .REJECTED:
+            // case (let jumioStatus, let nricStatus, let addressStatus) where jumioStatus == .REJECTED || nricStatus == .REJECTED || addressStatus == .REJECTED:
```

Comment:
Perhaps this comment might be clearer if you just delete up to `where` and only show the or'd logic (same below)

Reply:
sorry, meant to remove those comments!